x-pack/libbeat/cmd/inject.go

```diff
@@ -9,8 +9,13 @@ import (
 
```

Comment:
@urso It would be great if you could have especially a look at the things modified outside the agent directory.
@ph I'm wondering if we could put the changes outside the agent dir into a spearate PR to also backport them directly. Then when we release the agent in 7.x, the backport of agent is copy / pasting the directory over (mostly) and does not have other dependencies attached to it.

Reply:
I will close this one, and make smaller PRs

Reply:
+1 on separate PRs for non-agent modifications